core/field_dropdown.js

```diff
@@ -510,28 +510,6 @@ Blockly.FieldDropdown.prototype.renderSelectedText_ = function() {
   this.size_.width = Blockly.Field.getCachedWidth(this.textElement_);
 };
 
-/**
- * Updates the width of the field. Overrides field.prototype.updateWidth to
- * deal with image selections on IE and Edge. If the selected item is not an
- * image, or if the browser is not IE / Edge, this simply calls the parent
- * implementation.
- */
-Blockly.FieldDropdown.prototype.updateWidth = function() {
```

Comment:
Why did you remove this?

Reply:
It doesn't actually get called anywhere, all of the sizing logic is handled in the image and text render functions.